v11: fix for typescript definitions when using isolated modules (#1627)
emotion-js/emotion: .changeset/odd-insects-cough.md

---

> '@emotion/styled': minor

Reviewer: ```suggestion
'@emotion/styled': patch
```

---

> fix for typescript error importing @emotion/styled/base when using isolatedModules

Reviewer: ```suggestion
Fix for TypeScript error when importing `@emotion/styled/base` in combination with `isolatedModules` option.
```

Reviewer: I've wanted to commit this suggestion through the GitHub interface - but it seems like I don't have sufficient permissions to push to your fork.

You can check how to allow this for the future PRs here - https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork . At the moment just commit this suggestion with a click on a button.

Author: done, thanks
